tg.js

```diff
@@ -6,6 +6,7 @@ var nodeStatic = require('node-static');
 var mkdirp = require('mkdirp');
 var crypto = require('crypto');
 var nickcolor = require('./nickcolor');
+var myUser;
```

Comment:
I'm ok with this, other than that it might cause a very minor race condition. On line 176 where you compare to myUser.username there will be an error raised if myUser has not been initialized yet. This is because it's possible that a message is received before tg.getMe() is done.

To fix this I would suggest initializing myUser to an empty object here on line 9, so it would read

```
var myUser = {};
```

instead of what you have now. Then reading any member from the empty object will simply return undefined and not cause an error condition, causing your if condition to jump into the else block and getting the Telegram nick instead.